### ISSUE TITLE
10 years old empty file!

### ISSUE THREAD

**Issue by ale-rt** [plone/Products.CMFEditions#33]
Maybe this can be removed?
 - https://github.com/plone/Products.CMFEditions/blob/master/Products/CMFEditions/www/talesModifierAddForm.pt

Will the universe collapse?

**Comment by gforcada** [comment]
poor thing, let's give it a chance! (joking)

I think @svx has a script to check how old are the files on a given repository, it would be cool do that on certain packages and see how much +10 year old files we have around :smiley:

**Comment by svx** [comment]
Yes, I do have a script for this, but it is kind of 'beta' and needs some attention and cleaning, I use it sometimes on the core repositories and it tells you nice things like:
- $REPO is 10 years old
- last change on code was 2 weeks ago
- last update of /docs was 7 years ago
- last update of README was 4 years ago

I guess everyone can see on above output, what could be improved, right ? :)

**Comment by rafaelbco** [comment]
Seems very useful!

It would be cool  if you included in the output the file with the oldest
modification date. It would help to catch unused files.
Em 12/11/2015 8:10 AM, "sven" <notifications@github.com> escreveu:
